src/test/java/io/confluent/examples/streams/IntegrationTestUtils.java

```diff
@@ -339,10 +339,17 @@ private static boolean continueConsuming(final int messagesConsumed, final int m
     final Map<K, V> results = new LinkedHashMap<>();
     while (true) {
       final ProducerRecord<K, V> record = topologyTestDriver.readOutput(topic, keyDeserializer, valueDeserializer);
-      if (record == null) {
+      // Tables ignore records with null keys.
+      if (record == null || record.key() == null) {
         break;
       } else {
-        results.put(record.key(), record.value());
+        // For tables, a null-valued record represents a "DELETE" or tombstone for the corresponding key.
+        if (record.value() == null) {
+          results.remove(record.key());
+        }
+        else {
```

Comment:
nit: fix formatting -> should be only line above

Reply:
Ack.